mobile-app/app/screens/AppNavigator/screens/Portfolio/PortfolioScreen.tsx

```diff
@@ -455,6 +456,17 @@ export function PortfolioScreen ({ navigation }: Props): JSX.Element {
     ]
   }, [address, isLight])
 
+  const [tabButtonLabel, setTabButtonLabel] = useState<string>('')
+  const setButtonLabel = (buttonGroupTabKey: ButtonGroupTabKey): void => {
+    switch (buttonGroupTabKey) {
+      case (ButtonGroupTabKey.LPTokens):
+        return setTabButtonLabel('LP tokens')
```

Comment:
instead of using string as the value, is it possible if we make sure of the enum value? e.g. when defining enum, `ButtonGroupTabKey.LPTokens = 'LP tokens'`